pdf2image/pdf2image.py

```diff
@@ -26,12 +27,16 @@
 
 TRANSPARENT_FILE_TYPES = ['png', 'tiff']
 
-def convert_from_path(pdf_path, dpi=200, output_folder=None, first_page=None, last_page=None, fmt='ppm', thread_count=1, userpw=None, use_cropbox=False, strict=False, transparent=False, output_file=str(uuid.uuid4())):
+
+def convert_from_path(pdf_path, dpi=200, poppler_path=None, output_folder=None, first_page=None, last_page=None,
```

Comment:
Please add the new parameter last, while I understand that it's not as simply semantically, it prevents backward incompatible changes if users were not using the named parameters.